README.md

```diff
@@ -45,3 +45,11 @@ Production bundling with https://github.com/bazelbuild/rules_closure/ is
 underway, should be available Dec 2017 or Jan 2018
 
 Code-splitting and lazy loading are planned for Q1 2018
+
+Experimental Production bundling with Rollup:
+
+```bash
+$ bazel run src:rollup
```

Comment:
thinking out loud: we'll need to set users expectations about whether we'd ever support code splitting and lazy loading with a rollup setup. It's probably hard and not worth it since we'd be creating a new capability in that toolchain?
/cc @robwormald